apps/web/src/lib/components/popover/popover.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+  import flip from '@popperjs/core/lib/modifiers/flip';
+  import offset from '@popperjs/core/lib/modifiers/offset';
+  import { createPopper } from '@popperjs/core/lib/popper-lite';
+  import type { Instance } from '@popperjs/core';
+  import { tick } from 'svelte';
+  import { clickOutside } from '$lib/functions/use-click-outside';
+  import { browser } from '$app/env';
+  import { generateUUID } from '../../functions/util';
+  import { popovers } from './popover';
+
+  export let content = '';
+  export let singlePopover = true;
+
+  let contentElement: HTMLElement;
+  let iconElement: HTMLElement;
+  let popoverElement: HTMLElement;
+
+  let id: string;
+  let instance: Instance;
+  let isOpen = false;
+
+  $: if (browser) {
+    id = generateUUID(window);
```

Comment:
Actually I think you can use [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) instead of uuid for unique id